sdk/core/azure-core/test/ut/curl_session_test_test.cpp

```diff
@@ -136,6 +136,93 @@ namespace Azure { namespace Core { namespace Test {
         .clear();
   }
 
+  TEST_F(CurlSession, invalidHeader)
+  {
+    std::string response("HTTP/1.1 200 Ok\r\ninvalid header\r\n\r\nbody");
```

Comment:
This is an invalid header because of key:value pair being missing, right?

Reply:
Yes.